packages/client/hmi-client/src/assets/css/theme/designer/components/data/_datatable.scss

```diff
@@ -25,20 +25,23 @@
         border-width: $tableFooterBorderWidth;
         padding: $tableFooterPadding;
         font-weight: $tableFooterFontWeight;
+
     }
 
-    .p-datatable-thead > tr > th {
+    .p-datatable-thead>tr>th {
         text-align: $tableCellContentAlignment;
         padding: $tableHeaderCellPadding;
         border: $tableHeaderCellBorder;
         border-width: $tableHeaderCellBorderWidth;
         font-weight: $tableHeaderCellFontWeight;
+        font-size: $fontCaption;
         color: $tableHeaderCellTextColor;
         background: $tableHeaderCellBg;
         transition: $listItemTransition;
+        text-transform: uppercase;
     }
```

Comment:
What is the impact of this `uppercase` change? Do we have a screen shot? 
Asking because when you have math, equations and stuff when you make text uppercase it can mean a different thing than the lowercase counterpart.

Reply:
The uppercase just makes sure that the headers on the table are uppercase.

![image](https://user-images.githubusercontent.com/28237258/219064775-64753ba4-886c-4bff-857c-ffbca0d9ad88.png)

Reply:
got it, thanks 👍